### PR TITLE
Feature/39330

### PR DESCRIPTION
**Pull request by szafran89**
Internal: 39330

### CODE REVIEW

File: components/03-modules/cart-list-item/_cart-list-item.scss [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        flex-direction: column;
+        flex: 1;
+    }
+
+    &__data {
+        display: flex;
+        flex-direction: column;
+        @include mq($screen-m) {
+            flex-direction: row;
+            flex-flow: row wrap;
```
Comment (Aekal) [inline components/03-modules/cart-list-item/_cart-list-item.scss]: You can remove `flex-direction: row`, when `flex-flow` is in use

File: components/04-views/cart/cart.hbs [hunk 2]
```diff
@@ -1,95 +1,122 @@
...
+
+<div class="container">
+    {{ render '@heading' mainHeading }}
+</div>
+
+<div class="container cart">
+    <div class="grid">
+        <div class="grid__column grid__column--4-4">
+
+        </div>
```
Comment (Aekal) [inline components/04-views/cart/cart.hbs]: Please remove empty line

File: components/03-modules/cart-list-item/_cart-list-item.scss (hunk 1 above)
Comment (szafran89) [reply components/03-modules/cart-list-item/_cart-list-item.scss]: true, I omitted it

Comment (szafran89) [reply components/03-modules/cart-list-item/_cart-list-item.scss]: Done.

File: components/04-views/cart/cart.hbs (hunk 2 above)
Comment (szafran89) [reply components/04-views/cart/cart.hbs]: Done.

**Pull request merged**